packages/opentelemetry-node/lib/instrumentations.js

```diff
@@ -232,6 +235,16 @@ function getInstrumentations(opts = {}) {
             return;
         }
 
+        // Skip if metrics are disabled by env var
+        const isMetricsDisabled =
+            process.env.ELASTIC_OTEL_METRICS_DISABLED === 'true';
```

Comment:
nit: Not for this PR, but we should have a utility that parses envvars. FOr example, boolean env vars should not be case-sensitive.

Reply:
you're right. I'll create a ticket for it

Reply:
ref: https://github.com/elastic/elastic-otel-node/issues/441